### ISSUE TITLE
any offline support for it?

### ISSUE THREAD

**Issue by jack-solutions** [erickgnavar/transfer_products_app#1]
does it has any offline support and sync when online?

**Comment by erickgnavar** [comment]
Hi @jack-solutions, this repo is just a demo app.

**Comment by jack-solutions** [comment]
@erickgnavar so it seems like this app is not directly connecting to ODOO? it seems there is a middleware in between this mobile app and odoo that made in either PHP or nodejs or any other server technology.. am i correct?

**Comment by erickgnavar** [comment]
Yes, this app use a [web app](https://github.com/erickgnavar/transfer_products_app_backend) written in python that works as a bridge between the mobile app and Odoo.
